packages/app/app/stacks/components/StacksTable.tsx

```diff
@@ -0,0 +1,191 @@
+    availableBalance: null,
+    executedBuyAmount: "182201661599561308717",
+    executedSellAmount: "13783979663205039289",
+    executedSellAmountBeforeFees: "13783979663205039289",
+    executedFeeAmount: "0",
+    invalidated: false,
+    status: "fulfilled",
+    class: "limit",
+    surplusFee: "4855798329137440",
+    surplusFeeTimestamp: "2023-06-01T17:47:25.339953Z",
+    executedSurplusFee: "4855798329137440",
+    settlementContract: "0x9008d19f58aabd9ed0d60971565aa8510560ab41",
+    fullFeeAmount: "4855798329137440",
+    solverFee: "4855798329137440",
+    isLiquidityOrder: false,
+    sellToken: "0xe91d153e0b41518a2ce8dd3d7944fa863463a97d",
+    buyToken: "0x177127622c4a00f3d409b75571e12cb3c8973d3c",
+    receiver: "0x05a4ed2367bd2f0aa63cc14897850be7474bc722",
+    sellAmount: "13783979663205039289",
+    buyAmount: "1",
+    validTo: 1685643300,
+    appData:
+      "0x9b5c6dfa0fa4be89e17700f05bee8775b281aa6d2dac7dfbf3945e0f9642d777",
+    feeAmount: "0",
+    kind: "sell",
+    partiallyFillable: false,
+    sellTokenBalance: "erc20",
+    buyTokenBalance: "erc20",
+    signingScheme: "eip1271",
+    signature:
+      "0x000000000000000000000000e91d153e0b41518a2ce8dd3d7944fa863463a97d000000000000000000000000177127622c4a00f3d409b75571e12cb3c8973d3c00000000000000000000000005a4ed2367bd2f0aa63cc14897850be7474bc722000000000000000000000000000000000000000000000000bf4a8879188b7cb90000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000006478e0249b5c6dfa0fa4be89e17700f05bee8775b281aa6d2dac7dfbf3945e0f9642d7770000000000000000000000000000000000000000000000000000000000000000f3b277728b3fee749481eb3e0b3b48980dbbab78658fc419025cb16eee34677500000000000000000000000000000000000000000000000000000000000000005a28e9363bb942b639270062aa6bb295f434bcdfc42c97267bf003f272060dc95a28e9363bb942b639270062aa6bb295f434bcdfc42c97267bf003f272060dc9",
+    interactions: {
+      pre: [],
+      post: [],
+    },
+  },
+];
+
+// mock function to be replaced
+const getCowOrders = (order: Order) => mockCowOrders;
+
+// we'll probably use viem/ethers function when we connect to data
+const convertedAmount = (amount: string | number, decimals: number) =>
+  Number(amount) / 10 ** decimals;
+
+const calculateAveragePrice = (order: Order) => {
+  let totalExecutedBuyAmount = 0;
+  let totalExecutedSellAmount = 0;
+  getCowOrders(order).forEach((cowOrder) => {
+    if (cowOrder.executedBuyAmount === "0") return;
+
+    totalExecutedBuyAmount += convertedAmount(
+      cowOrder.executedBuyAmount,
+      order.buyToken.decimals
+    );
+    totalExecutedSellAmount += convertedAmount(
+      cowOrder.executedSellAmount,
+      order.sellToken.decimals
+    );
+  });
+  const averagePrice = totalExecutedSellAmount / totalExecutedBuyAmount;
+  return averagePrice;
+};
+
+export const ordersDone = (order: Order) => {
```

Comment:
```suggestion
export const totalOrdersDone = (order: Order) => {
```

Reply:
I this as:
`totalOrders` = total number of orders
`ordersDone` = number of orders done

Adding total doesn't make sense in my opinion.

Reply:
I suggested total in the sense total equals a number. `totalOrders` is good because I know it returns a number, but `ordersDone` seems to return the orders that are already done

Reply:
interesting, that makes sense, didn't see it that way. Thanks for explaining deeper.

Now I can see why `totalOrdersDone` make sense.

I'll change it to `totalOrdersDone`

Reply:
I have organised this on the Stack Modal PR so I'll change there if it is ok

Reply:
updated here: https://github.com/SwaprHQ/stackly-ui/pull/23/commits/f738ea744b0405f142fc55c1002e5d0e73db32d6